mentoris/templates/mentapp/login.html

```diff
@@ -54,23 +54,23 @@ <h2>Welcome, Please Log In!</h2>
                 {% csrf_token %}
                 <div class="mx-auto" style="width: 500px">
                     <input
-                        type="text"
+                        type="email"
                         class="form-control"
-                        id="username"
-                        name="username"
-                        placeholder="Username"
-                        autocomplete="username"
-                        value="{{ username }}"
+                        id="email"
+                        name="email"
+                        placeholder="Email Address"
+                        autocomplete="email"
+                        value="{{ email }}"
                         required
                     />
                     <div class="invalid-feedback">
-                        Please enter your username.
+                        Please enter your email.
```

Comment:
In the future will we allow them to enter their username to log in as well?

Reply:
Because we got rid of display name, probably not. Them entering their full name probably isn't enough to identify them (since more than one person could share the same full name).